packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -177,36 +176,19 @@ export class SpaceWidget extends Component {
       spaceId,
       sparkInstance,
       sparkState,
-      spark,
       toPersonEmail,
       toPersonId,
       mercury,
       metrics,
       user,
       spaceDetails,
-      widgetSpace,
       widgetStatus
     } = props;
-    const {formatMessage} = props.intl;
-
-    const registerErrorId = `spark.register`;
-    const toPerson = widgetSpace.get(`toPerson`);
-    if (sparkState.get(`registerError`) && (!errors.get(`hasError`) || !errors.get(`errors`).has(registerErrorId))) {
-      const error = spark.get(`error`);
-      let displaySubtitle = formatMessage(messages.unknownError);
-      if (error.statusCode === 401) {
-        displaySubtitle = formatMessage(messages.errorBadToken);
-      }
-      props.addError({
-        id: registerErrorId,
-        displayTitle: formatMessage(messages.unableToLoad),
-        displaySubtitle,
-        temporary: false,
-        code: error.statusCode
-      });
-    }
 
-    else if (sparkInstance && sparkState.get(`authenticated`) && sparkState.get(`registered`) && !sparkState.get(`hasError`)) {
+    if (sparkInstance
```

Comment:
Wonder if we should make a helper field for spark state that does these checks for us. Like `sparkState.status === 'ready'`

Reply:
Yea, I'm gonna try to integrate that into the refactor I'm doing.